src/corecel/sys/ScopedDeviceProfiling.cuda.cc

```diff
@@ -20,24 +20,38 @@
 
 namespace celeritas
 {
+namespace
+{
+nvtxDomainHandle_t domain_handle()
+{
+    static nvtxDomainHandle_t domain = nvtxDomainCreateA("Celeritas");
```

Comment:
Should we have the domain match the namespace casing (`celeritas`?) Or is it standard practice to have the domain fancy-cased?

Reply:
We can match the namespace